pkg/apis/storage/types.go

```diff
@@ -277,6 +277,14 @@ type CSIDriverSpec struct {
 	// +optional
 	AttachRequired *bool
 
+	// Defines if the underlying volume supports changing ownership and
+	// permission of the volume before being mounted.
+	// Refer to the specific FSGroupPolicy values for additional details.
+	// This field is alpha-level, and is only honored by servers
+	// that enable the CSIVolumeFSGroupPolicy feature gate.
```

Comment:
Mention the default here too

Reply:
Addressed in https://github.com/kubernetes/kubernetes/pull/93195